internal/load-test/single_msg_load.py

```diff
@@ -1,53 +1,22 @@
-import argparse, os, sys, time, logging
-from core.keys import keys_show
-from modules.auth.query import account_type, query_account
-from modules.bank.query import query_balances
-from modules.bank.tx import tx_send
-from utils import (
-    validate_num_txs,
-    print_balance_deductions,
+"""
+This script test a series of bank transfer transactions with single message between two accounts.
+It takes two optional arguments namely -s(sender) and -r(receiver).
+"""
+import sys
+from internal.core.parser import Parser
+from internal.stats.stats import clear_data_by_type, print_stats
+from internal.modules.auth.query import get_sequences
+from internal.modules.bank.query import calculate_balance_deductions, query_balances
+from internal.modules.bank.tx import tx_send
+
+p = Parser(
```

Comment:
`Parser` sounds like generalized, either it should parse any arguments or should be renamed to something specific to this requirement. How about `ParseTestsDefaultFlags`